http/src/main/resources/init-resources/cluster-site.conf

```diff
@@ -21,15 +25,15 @@
     RewriteCond %{REQUEST_URI} /proxy/[^/]*/[^/]*/rstudio/.* [NC]
     RewriteRule /proxy/[^/]*/[^/]*/rstudio/(.*) http://127.0.0.1:8001/$1 [P,L]
 
-    ProxyPass / http://127.0.0.1:8001/ retry=10
```

Comment:
Turns out we didn't need `ProxyPass` since we have `RewriteRule`s defined for each proxy path. 

`ProxyPassReverse` is needed for RStudio but not Jupyter.